codalab/client/local_bundle_client.py

```diff
@@ -167,9 +167,8 @@ def get_bundle_target(self, target):
         return (self.model.get_bundle(bundle_uuid), subpath)
 
     def get_worksheet_uuid(self, base_worksheet_uuid, worksheet_spec):
-        if worksheet_spec == '' or worksheet_spec == worksheet_util.HOME_WORKSHEET:
-            # Default worksheet name: take the username.
-            worksheet_spec = spec_util.home_worksheet(self._current_user_name())
+        if worksheet_spec == '' or worksheet_spec == worksheet_util.DASHBOARD:         
```

Comment:
As I mentioned in the codalab-worksheets PR, let's not replace HOME_WORKSHEET with DASHBOARD completely.  '' and '/' should still refer to the HOME_WORKSHEET